sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_generated/_configuration.py

```diff
@@ -45,7 +45,11 @@ def __init__(self, endpoint: str, credential: Union[AzureKeyCredential, "TokenCr
         self.endpoint = endpoint
         self.credential = credential
         self.api_version = api_version
-        self.credential_scopes = kwargs.pop("credential_scopes", ["https://azconfig.io/.default"])
+        credential_scopes = kwargs.pop("credential_scopes", ["https://azconfig.io/.default"])
```

Comment:
This file is in a folder called generated. That sounds like something that isn't meant to have manual touches, no?